taker/src/routes.rs

```diff
@@ -134,7 +135,8 @@ pub async fn feed(
                 }
                 Ok(()) = rx_quote.changed() => {
                     let quote = rx_quote.borrow().clone();
-                    yield quote.to_sse_event();
+                    yield Event::json(&quote.get(&model::ContractSymbol::BtcUsd)).event("btcusd_quote");
```

Comment:
Looks like these events are not consumed by the taker-ui. Maybe we don't have to emit them then?

Reply:
I don't mind disabling them altogether; I'll probably do that explicitly in a separate commit on top, so that it can be reverted if we ever change our minds about this.

Reply:
removed quote SSE events from the taker side altogether.